pkg/service/v0/service.go

```diff
@@ -30,8 +31,15 @@ func NewService(opts ...Option) Service {
 		mux:    m,
 	}
 
-	m.HandleFunc("/v1.0/me", svc.Me)
-	m.HandleFunc("/v1.0/users", svc.Users)
+	m.Route("/v1.0", func(r chi.Router) {
+		r.Get("/me", svc.Me)
+		r.Route("/users", func(r chi.Router) {
+			r.Get("/", svc.Users)
+			r.Route("/{userId}", func(r chi.Router) {
+				r.Get("/", svc.Users)
```

Comment:
It's more common to have a separate handler for that, just add the definition to the interface and all service implementations.

Reply:
Sure - I'll add some more refactorings .... need to extract all ldap stuff out of service.go